management-web/src/main/java/com/ns/task/product/ProductController.java

```diff
@@ -0,0 +1,47 @@
+package com.ns.task.product;
+
+import com.mx.task.model.ProductDTO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import javax.validation.Valid;
+
+@Controller
+public class ProductController {
+
+
+    private final ProductService productService;
+
+    @Autowired
+    public ProductController(ProductService productService) {
+        this.productService = productService;
+    }
+
+    @GetMapping("/addProductForm")
+    public String showAddProductForm(ModelMap model) {
+        model.put("product", new ProductDTO());
+
+        return "addProductForm";
+    }
+
+    @GetMapping("/products")
+    public String showProductsPage(ModelMap model) {
+        model.put("products", productService.getProducts());
+        return "products";
+    }
+
+    @PostMapping("/addProductForm")
```

Comment:
The URI should follow the REST API conventions, in this case, I think you can have it as `addProduct` or just `product`